### PR TITLE
 [proxima-cassandra] user attribute name for construction of CQL when…

### PR DESCRIPTION
**Pull request by je-ik**
… missing payloadCol

### CODE REVIEW

File: cassandra/src/main/java/cz/o2/proxima/storage/cassandra/DefaultCQLFactory.java [hunk 1]
```diff
@@ -321,4 +322,7 @@ public Statement scanPartition(
     return new SimpleStatement(query);
   }
 
+
+
+
```
Comment (dmvk) [inline cassandra/src/main/java/cz/o2/proxima/storage/cassandra/DefaultCQLFactory.java]: 👎

File: cassandra/src/main/java/cz/o2/proxima/storage/cassandra/CacheableCQLFactory.java [hunk 2]
```diff
@@ -294,6 +296,13 @@ protected String toUnderScore(String what) {
     return sb.toString();
   }
 
+  String toPayloadCol(AttributeDescriptor<?> attr) {
```
Comment (dmvk) [inline cassandra/src/main/java/cz/o2/proxima/storage/cassandra/CacheableCQLFactory.java]: can you add a short comment about what the method does and what payloadCol is?

File: cassandra/src/main/java/cz/o2/proxima/storage/cassandra/DefaultCQLFactory.java (hunk 1 above)
Comment (je-ik) [reply cassandra/src/main/java/cz/o2/proxima/storage/cassandra/DefaultCQLFactory.java]: Oops. :)

**Comment by je-ik** [comment]
Fixed. Thanks for review!

**Pull request merged**